content/docs/command-reference/remote/modify.md

```diff
@@ -271,6 +271,43 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
   $ dvc remote modify myremote gdrive_client_secret <client secret>
   ```
 
+- `gdrive_use_service_account` - instructs DVC to auth with a
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account).
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_use_service_account <True|False>
+  ```
+
+- `gdrive_service_account_email` - Google Project's
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account)
+  email address.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_email <service account email>
+  ```
+
+- `gdrive_service_account_p12_file_path` - Google Project's
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-gdrive-remote-with-a-service-account)
+  `.p12` file path.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_p12_file_path <file_path.p12>
+  ```
+
+- `gdrive_service_account_user_email` - user email to be impersonated by
```

Comment:
```suggestion
- `gdrive_service_account_user_email` - user email corresponding to the
```

Reply:
`impersonate some user account` is wide used term for Google Service Accounts mentioned in official docs as well

Reply:
Interesting. OK, how about

```suggestion
- `gdrive_service_account_user_email` - email of the service account to impersonate
```

I'm a little confused about the term "user" here and not sure how/whether to include it. Is there a user account AND a service account (owned by the user) AND a 3rd account that will impersonate?

Reply:
Updated. Moved Q about terminology to https://github.com/iterative/dvc/commit/3ff804b3b28abe67ffa206b92c8345acb61935dd#r38129539.